src/quo2/components/record_audio/record_audio/buttons/record_button_big.cljs

```diff
@@ -45,129 +45,131 @@
+                  (reset! recording-length-ms 0)
+                  (log/debug "[record-audio] stop recording - success"))
+                #(log/error "[record-audio] stop recording - error: " %))))
+           start-animation (fn []
+                             (set-value opacity 1)
+                             (animate-linear scale 2.6 signal-anim-duration)
+                             ;; TODO: Research if we can implement this with withSequence method
+                             ;; from Reanimated 2
+                             ;; GitHub issue [#14561]:
+                             ;; https://github.com/status-im/status-mobile/issues/14561
+                             (reset! clear-timeout
+                               (js/setTimeout
+                                (fn []
+                                  (set-value scale scale-to-each)
+                                  (animate-linear-with-delay-loop scale
+                                                                  scale-to-total
+                                                                  signal-anim-duration-2
+                                                                  0))
+                                signal-anim-duration)))
+           stop-animation (fn []
+                            (set-value opacity 0)
+                            (reanimated/cancel-animation scale)
+                            (set-value scale 1)
+                            (when @clear-timeout (js/clearTimeout @clear-timeout)))
+           start-y-animation (fn []
+                               (reset! record-button-at-initial-position? false)
+                               (reset! record-button-is-animating? true)
+                               (animate-easing translate-y -64 250)
+                               (animate-linear-with-delay icon-opacity 0 33.33 76.66)
+                               (js/setTimeout (fn []
+                                                (reset! record-button-is-animating? false)
+                                                (when-not @touch-active? (complete-animation)))
+                                              250))
+           reset-y-animation (fn []
+                               (animate-easing translate-y 0 300)
+                               (animate-linear icon-opacity 1 500)
+                               (js/setTimeout (fn []
+                                                (reset! record-button-at-initial-position? true))
+                                              500))
+           start-x-animation (fn []
+                               (reset! record-button-at-initial-position? false)
+                               (reset! record-button-is-animating? true)
+                               (animate-easing translate-x -64 250)
+                               (animate-linear-with-delay icon-opacity 0 33.33 76.66)
+                               (animate-linear red-overlay-opacity 1 33.33)
```

Comment:
Can we have the numbers in these animations get some defintions and/or equations? For example 0s and 1s are probably initial values for something. Translating-x by `-64` this may be based on some other variable of a container. If the button's and container's styling changed, are these numbers still going to work?

An example file to show what I mean: https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/chat/lightbox/zoomable_image/view.cljs

Reply:
We can for sure, but we have many files that are part of the whole component and probably would be better to open a separate PR for refactoring it. Doing it in this one and only in this file would be incomplete and also would distort the reason for this PR.

The only relevant changes in this file should be in lines 85 and 86, but the linter modified bunch more of lines when running `make lint-fix`. So I'd suggest to make refactors separately.

Reply:
Sure no problem